packages/client/modules/userDashboard/components/OrgBilling/OrgFeatures.tsx

```diff
@@ -34,23 +40,49 @@ const FeatureNameGroup = styled('div')({
   }
 })
 
-const OrgFeatures = () => {
-  const [showAIFeatures, setShowAIFeatures] = useState(false)
+interface Props {
+  organizationRef: OrgFeatures_organization$key
+}
+
+const OrgFeatures = (props: Props) => {
+  const {organizationRef} = props
+  const atmosphere = useAtmosphere()
+  const {onError, onCompleted} = useMutationProps()
+  const organization = useFragment(
+    graphql`
+      fragment OrgFeatures_organization on Organization {
+        id
+        isOrgAdmin
+        useAI
+      }
+    `,
+    organizationRef
+  )
+  const {id: orgId, isOrgAdmin, useAI} = organization
+
+  const handleToggle = () => {
+    const variables = {orgId}
+    ToggleAIFeaturesMutation(atmosphere, variables, {
```

Comment:
+1 if you use the `useMutation` pattern instead of the older pattern then you don't need `useMutationProps` in this component & it'll be a little cleaner